GetStream/stream-chat-android
# [43] Migrate guides to the main repo

### 🎯 Goal

https://github.com/GetStream/android-internal-board/issues/43

We decided to migrate only guides from https://github.com/GetStream/Android-Samples to the main repo. 

### 🛠 Implementation details

Merged the guides into one application to avoid code duplication and multiple launch targets.

### 🎨 UI Changes

https://user-images.githubusercontent.com/9600921/184360236-6d404fad-2d9e-4f10-aa5c-0d87cbe226a8.mp4

### 🧪 Testing

1. Run `stream-chat-android-ui-guides` app.
2. Check the guides

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
